recipes/pybufr-ecmwf/meta.yaml

```diff
@@ -3,46 +3,40 @@
 {% set sha2563k = "b4027e018188a1ea29ab3fb5374087bac569c290335a1d5cde8356b446069435" %}
 
 package:
-  name: pybufr-ecmwf  # [py2k]
-  name: pybufr-ecmwf3  # [py3k]
```

Comment:
We cannot create a feedstock with multiple names now with a name that does not match the recipe directory. In theory we do not need to append the `3` because conda does not allow to install multiple versions of Python in the same env.

I do understand that the original source has the `3` in the and, even though I do not like that, we should maybe try to honor it. One way is to add only the `Python 2` version now and once the `feedstock` is created we can rename it without the constraints I mentioned above.